tools/dht_put.cpp

```diff
@@ -73,7 +74,7 @@ void usage()
 	exit(1);
 }
 
-alert* wait_for_alert(lt::session& s, int alert_type)
```

Comment:
there's actually a name clash here. in older versions of osx (and presumably bsd) there's a C function called session() (or maybe it's a struct) that gets pulled in via some system header. That's where this came from